h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1369,6 +1410,11 @@ private int run2(String[] args) throws Exception {
       addMapperArg(conf, s);
     }
 
+    if (client) {
+      addMapperArg(conf, "-md5skip");
```

Comment:
Idea: We should probably skip md5skip for clients, and have better hand-shake (e.g., baed on major H2O version).